glue/sales_val_flagging.py

```diff
@@ -759,6 +763,11 @@ def write_to_table(df, table_name, s3_warehouse_bucket_path, run_id):
     )
     """
 
+    SQL_QUERY_SALES_VAL = f"""
+    SELECT *
+    FROM {args["sale_flag_table"]}
+    """
```

Comment:
**[Question, non-blocking]** I think it's not necessary to revert the section of #53 that removed this code block and altered the subsequent code to avoid a duplicate query, right? It seems to me that it's unrelated to the `sale.flag` join but I could be wrong 🤔